lib/execution_engine2/execution_engine2Impl.py

```diff
@@ -9,7 +9,7 @@
 from execution_engine2.utils.APIHelpers import GenerateFromConfig
 from execution_engine2.utils.clients import get_client_set
 
-
+_AS_ADMIN = "as_admin"
```

Comment:
Only the 3 `as_admin` related lines are actual code changes, the rest of the lines are from recompiling.